tests/Integration/Behaviour/Features/Context/Domain/AttributeGroupFeatureContext.php

```diff
@@ -56,7 +56,7 @@ public function createAttributeGroup(string $reference, TableNode $node): void
     {
         $data = $this->localizeByRows($node);
 
-        $attributeGroupId = $this->createAttributeGroupUsingCommand($data['name'], $data['public_name'], $data['type']);
+        $attributeGroupId = $this->createAttributeGroupUsingCommand([1 => $data['name']], [1 => $data['public_name']], $data['type']);
```

Comment:
well it was failing because it didn't use the CQRS command, and the expected format was not the same, I'll see how I can have it working with previous test implementation and current CQRS command

Reply:
You should use multi-lang values like `name[en-US]` like in many other behat tests, coupled with the `localizeByRows` and similar methods this should be doable

Reply:
This still doesn't rely on localized values 🤔